data/scripts.js

```diff
@@ -619,11 +627,11 @@ let BattleScripts = {
 			move.hit = hit;
 			if (move.smartTarget && targets.length > 1) {
 				targetsCopy = [targets[hit - 1]];
-				if (targetsCopy[0] && hit > 1) this.addMove('-anim', pokemon, move.name, targetsCopy[0]);
 			} else {
 				targetsCopy = targets.slice(0);
 			}
 			let target = targetsCopy[0]; // some relevant-to-single-target-moves-only things are hardcoded
+			if (target && typeof move.smartTarget === 'boolean') this.addMove('-anim', pokemon, move.name, target);
```

Comment:
This should still use `retargetLastMove` for the first hit, instead of the `[still]` hack...

Reply:
bump